packages/runner-ct/webpack.config.ts

```diff
@@ -1,101 +0,0 @@
-process.env.NO_LIVERELOAD = '1'
-
-import _ from 'lodash'
-import path from 'path'
-import type webpack from 'webpack'
-import { getCommonConfig, getCopyWebpackPlugin } from '@packages/web-config/webpack.config.base'
-import * as cyIcons from '@packages/icons'
-
-const commonConfig = getCommonConfig()
-const CopyWebpackPlugin = getCopyWebpackPlugin()
-
-// @ts-ignore
-const babelLoader = _.find(commonConfig.module.rules, (rule) => {
-  // @ts-ignore
-  return _.includes(rule.use.loader, 'babel-loader')
-})
-
-// @ts-ignore
-babelLoader.use.options.plugins.push([require.resolve('babel-plugin-prismjs'), {
-  languages: ['javascript', 'coffeescript', 'typescript', 'jsx', 'tsx'],
-  plugins: ['line-numbers', 'line-highlight'],
-  theme: 'default',
-  css: false,
-}])
-
-const { pngRule, nonPngRules } = commonConfig!.module!.rules!.reduce<{
-  nonPngRules: webpack.RuleSetRule[]
-  pngRule: webpack.RuleSetRule | undefined
-}>((acc, rule) => {
-  if (rule?.test?.toString().includes('png')) {
-    return {
-      ...acc,
-      pngRule: rule,
-    }
-  }
-
-  return {
-    ...acc,
-    nonPngRules: [...acc.nonPngRules, rule],
-  }
-}, {
-  nonPngRules: [],
-  pngRule: undefined,
-})
-
-if (!pngRule || !pngRule.use) {
-  throw Error('Could not find png loader')
-}
-
-(pngRule.use as webpack.RuleSetLoader[])[0].options = {
-  name: '[name].[ext]',
-  outputPath: 'img',
-  publicPath: '/__cypress/runner/img/',
-}
-
-// @ts-ignore
-const config: webpack.Configuration = {
-  ...commonConfig,
-  module: {
-    rules: [
-      ...nonPngRules,
-      pngRule,
-      {
-        test: /index\.js/,
-        exclude: /node_modules/,
-      },
-    ],
-  },
-  entry: {
-    cypress_runner: [path.resolve(__dirname, 'src/index.js')],
-  },
-  output: {
-    path: path.resolve(__dirname, 'dist'),
-    filename: '[name].js',
-    chunkFilename: '[name].[chunkhash].js',
```

Comment:
I want to call out this is the key / largest difference in webpack configuration difference between this and runner/webpack.config. I'll defer to the CT team if this is something that matters. if it does, I'd assume maybe here: https://github.com/cypress-io/cypress/pull/23028/files#diff-9af0f8f4f669446066e3dca23bb2645cb4f94dc9b11179a63f38cc657b36b30eR51

though assuming if tests are passing we are okay? cc// @lmiller1990 -- tagging you since you've already reviewed 😄

Reply:
Yeah, I don't think we need this. We had some weird situation going on where we would conditionally load the react devtools, which we tried to embed in the runner-ct (which would be cool, if it worked as expected) but that experiment is long gone.

It sure would be nice to have the devtools embedded in the runner automatically... the main problem is the react devtools are designed to be used for 1 app and not torn down unless you refresh the page or via hot reload. When you run 100s of CT tests, we continually tear down/reset in AUT (and effectively the React app) every couple of seconds. Doing this leads to some situations where the devtools show stale components, or worse, keep a reference to components that no longer exist ... which can give us a memory leak.